pkg/cmd/server/start/start_master.go

```diff
@@ -87,9 +89,9 @@ func NewCommandStartMaster() (*cobra.Command, *MasterOptions) {
 			if err := options.StartMaster(); err != nil {
 				if kerrors.IsInvalid(err) {
 					if details := err.(*kerrors.StatusError).ErrStatus.Details; details != nil {
-						fmt.Fprintf(c.Out(), "Invalid %s %s\n", details.Kind, details.ID)
+						fmt.Fprintf(out, "Invalid %s %s\n", details.Kind, details.ID)
```

Comment:
> Then what if the struct is nil?

Bare structs are never nil in Go.  They get an empty value, which is valid for a method receiver.